src/components/graphs/split-graph/split-graph.styles.ts

```diff
@@ -8,7 +8,8 @@ export const progressBarStyles = (
   bg: color,
   width: `${percent}%`,
   marginRight: '2px',
-  border: isSelected ? '1px solid #272727' : null,
+  border: isSelected ? '1px solid' : null,
+  borderColor: 'black',
```

Comment:
this is a pattern you're gonna see a lot: `borderColor` is the only way to use theme colors in borders :(